glide.yaml

```diff
@@ -48,22 +48,22 @@ import:
   - package: github.com/BurntSushi/toml
     version: ~0.3.0
   - package: github.com/prometheus/client_golang
-    version: 0.8.0
+    version: 0.9.2
   - package: github.com/grpc-ecosystem/go-grpc-prometheus
   - package: k8s.io/kubernetes
     version: release-1.14
```

Comment:
Is there a reason to tracking the `release-1.14` branch and not pin this to the `v1.14.1` tag?